From 21c23860165de0e39979dbb747c031a85cda1cc6 Mon Sep 17 00:00:00 2001
From: Gustavo Padovan <gustavo.padovan@collabora.co.uk>
Date: Thu, 21 Jan 2016 10:49:22 -0200
Subject: [PATCH 15/88] staging/android: remove unused var from
 sync_timeline_signal()
Git-commit: e6a0cb4819d082b60ce47e432bfcac16fdf93767
Patch-mainline: v4.6-rc1
References: fate#321223

signaled_pts is not used in this function.

Signed-off-by: Gustavo Padovan <gustavo.padovan@collabora.co.uk>
Reviewed-by: Maarten Lankhorst <maarten.lankhorst@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Patrik Jakobsson <pjakobsson@suse.de>
---
 drivers/staging/android/sync.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/android/sync.c b/drivers/staging/android/sync.c
index 3c2c8d07df62..9ec55ef16eb1 100644
--- a/drivers/staging/android/sync.c
+++ b/drivers/staging/android/sync.c
@@ -101,7 +101,6 @@ EXPORT_SYMBOL(sync_timeline_destroy);
 void sync_timeline_signal(struct sync_timeline *obj)
 {
 	unsigned long flags;
-	LIST_HEAD(signaled_pts);
 	struct fence *fence, *next;
 
 	trace_sync_timeline(obj);
-- 
2.12.0
